Questions

```diff
@@ -0,0 +1,7 @@
+I would like to indicate that there is an error when the code gets to the line 60. 
+
+print (tweets_data([0])
```

Comment:
I might have to look into again, since few libraries have changed over the years